src/main/java/org/jasig/portlet/announcements/model/validators/ValidationHelper.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.jasig.portlet.announcements.model.validators;
+
+/**
+ * Utility class for validation efforts
+ */
+public class ValidationHelper {
+  private static enum SpecialCharacters {
+    CONTROL1("\u00E2\u0080", ""), //Looks to always be together when conversion issues arise.
```

Comment:
:question: Could Guava's char matcher help manage some of these patterns?
https://github.com/google/guava/wiki/StringsExplained#charmatcher

Reply:
Hi @ChristianMurphy - It does look promising / more comprehensive for at least the control characters.  I'll have a look shortly and see how it handles my test scenarios.

Reply:
Sounds good, I'll make that change.

Reply:
@ChristianMurphy - I tried out CharMatcher javaIsoControl and it removes more characters then I wanted, so for now I recommended we keep this solution.